{{cookiecutter.project_name}}/contracts/NFT.vy

```diff
@@ -301,6 +317,49 @@ def royaltyInfo(_tokenId: uint256, _salePrice: uint256) -> (address, uint256):
     return self.owner, royalty
 {%- endif %}
 
+{%- if cookiecutter.force_royalties == 'y' %}
+# Helper functions in case market place does not support royalties to execute by this contract with _deductRoyalties()
+@internal
+@view
+def _royaltyInfo(_tokenId: uint256, _salePrice: uint256) -> (address, uint256):
+    """
+    /// @notice Called with the sale price to determine how much royalty
+    //          is owed and to whom. Important; Not all marketplaces respect this, e.g. OpenSea
+    /// @param _tokenId - the NFT asset queried for royalty information
+    /// @param _salePrice - the sale price of the NFT asset specified by _tokenId
+    /// @return receiver - address of who should be sent the royalty payment
+    /// @return owner address and royaltyAmount - the royalty payment amount for _salePrice
+    """
+
+    royalty: uint256 = convert(convert(_salePrice, decimal) * ROYALTY_TO_APPLY_TO_PRICE, uint256) # Percentage that accepts decimals
+    return self.owner, royalty
```

Comment:
and then here do:
```suggestion
    # NOTE: We are returning `self` as the receiver of the royalty to enforce payment to `self.owner` later
    return self, max(self.minRoyaltyAmount, royalty)

```

Reply:
@fubuloubu this `max` is a genius solution for a future implementation of a minimum sell price looking at `minRoyaltyAmount`. Hats off. 🙏